packages/sources/alphavantage-test/src/config/index.ts

```diff
@@ -0,0 +1,16 @@
+export const DEFAULT_API_ENDPOINT = 'forex'
```

Comment:
We probably don't need this line anymore from v2

Reply:
Done